src/main/java/hudson/plugins/analysis/collector/AnalysisGraphConfiguration.java

```diff
@@ -77,7 +77,7 @@ public boolean getCanDeacticateOtherTrendGraphs() {
 
     @Override
     public String serializeToString() {
-        return super.serializeToString() + SEPARATOR + serializeBoolean(canDeactivateGraphs);
+        return super.serializeToString() + serializeBoolean(canDeactivateGraphs);
```

Comment:
From 5ddfe743d71b76f898a75f83db4d265f5400c8f5. Seems like something that should have been done in a separate PR, merged into this one.